scvi/inference/jvae_trainer.py

```diff
@@ -155,7 +162,7 @@ def on_training_loop(self, tensors_list):
             self.optimizer.step()
 
         # Train generative model
-        self.current_loss = g_loss = self.loss(tensors_list)
+        self.current_loss = g_loss = self.loss(tensors_dict)
```

Comment:
I'm always worried of those double equals. Can we avoid this? Or are we sure it's good practice?

Reply:
double equal is probably okay for now, but i think all of this will be refactored when posterior is refactored